sdk/storage/azure-storage-common/inc/azure/storage/common/storage_common.hpp

```diff
@@ -50,10 +50,10 @@ namespace Azure { namespace Storage {
     HashAlgorithm Algorithm = HashAlgorithm::Md5;
   };
 
-  namespace _detail {
+  namespace _internal {
```

Comment:
Is there anything in storage common that is private to the package and belongs in `_detail` or not being used at all, that we should remove?

Reply:
I don't think there's anything in common package is private to this package.